hytra/core/ilastikhypothesesgraph.py

```diff
@@ -120,7 +127,24 @@ def getTransitionFeaturesRF(self, traxelA, traxelB, transitionClassifier, probab
         feats = [probabilityGenerator.getTraxelFeatureDict(obj.Timestep, obj.Id) for obj in [traxelA, traxelB]]
         featVec = probabilityGenerator.getTransitionFeatureVector(feats[0], feats[1], transitionClassifier.selectedFeatures)
         probs = transitionClassifier.predictProbabilities(featVec)[0]
-        return [probs[0]] + [probs[1]] * (max_state - 1)
+
+        # or image borders, so predict probability just by distance
+        upperBound = self.fieldOfView._FieldOfView__upperBound
+        lowerBound = self.fieldOfView._FieldOfView__lowerBound
```

Comment:
it would be nicer to add a getter for upper and lower bound of the field of view than to get hacky access to private members :)